Common/Tools/handleParamTPCResponse.cxx

```diff
@@ -0,0 +1,300 @@
+
+int main(int argc, char* argv[])
+{
+  bpo::options_description options("Allowed options");
+  bpo::variables_map arguments;
+  if (!initOptionsAndParse(options, argc, argv, arguments)) {
+    return 1;
+  }
+
+  std::unique_ptr<Response> tpc = nullptr;
+
+  const std::string urlCCDB = arguments["url"].as<std::string>();
+  const std::string pathCCDB = arguments["ccdb-path"].as<std::string>();
+  long startTime = arguments["start"].as<long>();
+  long endTime = arguments["stop"].as<long>();
+  const auto runnumber = arguments["runnumber"].as<unsigned int>();
+  auto timestamp = arguments["timestamp"].as<long>();
+  const int optDelete = arguments["delete-previous"].as<int>();
+
+  const std::string outFilename = arguments["save-to-file"].as<std::string>();
+  const std::string inFilename = arguments["read-from-file"].as<std::string>();
+  const std::string objname = arguments["objname"].as<std::string>();
+  const std::string inobjname = arguments["inobjname"].as<std::string>();
+
+  const float bb0 = arguments["bb0"].as<float>();
+  const float bb1 = arguments["bb1"].as<float>();
+  const float bb2 = arguments["bb2"].as<float>();
+  const float bb3 = arguments["bb3"].as<float>();
+  const float bb4 = arguments["bb4"].as<float>();
+  const std::string pathResoParam = arguments["reso-param-path"].as<std::string>();
+  const std::string sigmaGlobal = arguments["sigmaGlobal"].as<std::string>();
+  const float mipval = arguments["paramMIP"].as<float>();
+  const float chargefacval = arguments["paramChargeFactor"].as<float>();
+  const float multNormval = arguments["paramMultNormalization"].as<float>();
+  const bool useDefaultParam = arguments["useDefaultParam"].as<bool>();
+  const std::string optMode = arguments["mode"].as<std::string>();
+  if (optMode.empty()) {
+    LOG(error) << "--mode must be specified (read, write, pull, push)";
+    return 1;
+  }
+  // create parameter arrays from commandline options
+  std::array<float, 5> BBparams = {bb0, bb1, bb2, bb3, bb4};
+
+  std::vector<double> sigparamsGlobal;
+
+  if (pathResoParam != "") { // Read resolution parameters from file
+    std::ifstream infile(pathResoParam);
+    std::string paramstring;
+    std::vector<double> sigmaParams;
+    std::getline(infile, paramstring);
+    std::istringstream ss(paramstring);
+    sigmaParams.push_back({});
+    double param = 0.;
+    while (ss >> param) {
+      sigmaParams.push_back(param);
+    }
+    sigparamsGlobal = sigmaParams;
+  } else {
+    sigparamsGlobal = o2::RangeTokenizer::tokenize<double>(sigmaGlobal);
+  }
+
+  std::map<std::string, std::string> metadata, headers;
+  o2::ccdb::CcdbApi api;
+  if (optMode.compare("push") == 0 || optMode.compare("pull") == 0) { // Initialise CCDB if in push/pull mode
+    api.init(urlCCDB);
+    if (!api.isHostReachable()) {
+      LOG(warning) << "CCDB mode (push/pull) enabled but host " << urlCCDB << " is unreachable.";
+      return 1;
+    }
+    // Init timestamps
+    if (runnumber != 0) {
```

Comment:
This looks like inspired from https://github.com/AliceO2Group/O2Physics/blob/master/Common/Tools/handleParamBase.h#L82 is there something wrong/missing with it that it needed to be reimplemented?